stripe-buffer.el

```diff
@@ -136,9 +174,25 @@ Used by `stripe-table-mode' Only the first matching group will be painted."
                                     (goto-char end)
                                     (point))))))
                       ( overlay (apply get-overlay-create-function stripe-region)))
-                 (overlay-put overlay 'face stripe-highlight-face)
-                 (overlay-put overlay 'is-stripe t)
-                 (push overlay sb/overlays)))))
+
+                 (cond
+                  ((unless (require 'dired-subtree nil t)
```

Comment:
I understood what you say. Thanks. 
I commit https://github.com/myuhe/stripe-buffer/commit/aa139e24cc0a59c4882418ff3b3ffabd3e17a636 that fix it.

See it.